test/integration/controllers/category.spec.js

```diff
@@ -123,7 +130,43 @@ describe('Category controller', () => {
 
       expect(response.statusCode).toBe(200)
       expect(Array.isArray(response.body)).toBeTruthy()
-      expect(typeof response.body[0]).toBe('string')
+      expect(typeof response.body[0]).toBe('object')
+    })
+  })
+
+  describe(`shoud return min and mix prices ${endpointUrl}`, () => {
+    it('should throw OFFER_NOT_FOUND', async () => {
+      const response = await app
+        .get(endpointUrl + `${categoryData._id}/price-range?authorRole=student`)
+        .set('Authorization', `Bearer ${accessToken}`)
+
+      expectError(404, NOT_FOUND, response)
+    })
+
+    it('should return min and max prices for student offers', async () => {
+      const response = await app
+        .get(endpointUrl + `${categoryData._id}/subjects/${testSubject._id}/price-range?authorRole=student`)
+        .set('Authorization', `Bearer ${accessToken}`)
+
+      expect(response.statusCode).toBe(200)
+      expect(response.body && typeof response.body === 'object').toBe(true)
```

Comment:
maybe it's better to change it to something like expect(typeof response.body).toBe('object')? Seems like the result will be the same

Reply:
Fixed.